### PR TITLE
Fix/override prop UUID

### PR DESCRIPTION
**Pull request by ivan-015**

### CODE REVIEW

File: lib/interpret_and_optimize/services/pb_shared_aggregation_service.dart [hunk 1]
```diff
@@ -43,9 +47,14 @@ class PBSharedInterAggregationService {
...
       PBSharedMasterNode sharedMasterNode, PBIntermediateNode rootChildNode) {
     for (var prop in sharedMasterNode.overridableProperties) {
+      var targetUUID = _findLastOf(prop?.UUID, '/');
       prop.value = PBIntermediateNodeSearcherService.searchNodeByUUID(
-          rootChildNode, prop?.UUID);
-      if (prop.type == PBSharedInstanceIntermediateNode) {
+          rootChildNode, targetUUID);
+      if (prop.value == null) {
+        // add Designer Warning here, not even sure if this is the designers fault or not
+        // log.warning('UUID: ${targetUUID} not found in searchNodeByUUID');
```
Comment (SushiRoll53) [inline lib/interpret_and_optimize/services/pb_shared_aggregation_service.dart]: I think we should uncomment this log for now, so we know when it happens again. Then when the refactor hits, we can decide to which part of the logs should go.

**Review by SushiRoll53 — COMMENTED** [review]
What is the reason we need those .sketch files? 🤔

**Comment by ivan-015** [comment]
> What is the reason we need those .sketch files? 🤔

The files were added by Ken, they provide a more complete and robust Sketch file to test Parabeac-Core's output.

**Review by SushiRoll53 — APPROVED** [review]
LGTM

**Pull request merged**